keyboards/handwired/horrortroll/k552/readme.md

```diff
@@ -0,0 +1,14 @@
+# Redragon K552 Kumara
+
+![K552](https://i.imgur.com/XpNmJpB.png)
```

Comment:
```suggestion
![K552](https://i.imgur.com/XpNmJpBh.png)
```

Reply:
Umm, the link for image was correct. Is it something did u mistypo on it?

Reply:
It's the same image, just smaller.

Reply:
Ahh okay 👍